tests/modules/infermedicaAnalytics.test.js

```diff
@@ -0,0 +1,93 @@
+/* global __analytics */
+jest.useFakeTimers('modern');
+jest.setSystemTime(new Date(2020, 3, 1));
+let analyticModules;
+let publishPayload;
+const testProperties = { test: 'test', test2: 'test2', test3: 'test3' };
+const expectedPayload = {
+  events: [{
+    attributes: { environment: undefined },
```

Comment:
R: please add test for enviroment `__analytics.infermedicaAnalytics.environment = 'test';`